### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by Giyanstudio** [community/community#67606] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

![Screenshot 2023-09-19 182836](https://github.com/community/community/assets/145269051/505eef88-e0e0-4101-8d19-edc07bdad84e)

**Comment by queenofcorgis** [comment]
Hi @Giyanstudio , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.
